vendor/stormlib-9.31/CMakeLists.txt

```diff
@@ -0,0 +1,116 @@
+# StormLib 9.31 - CMake build for integration with squall
+
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Since this is being included from the root-level `CMakeLists.txt`, I don't think we need to declare specific compatibility here.